Bio/Seq.py

```diff
@@ -28,7 +28,8 @@
 from Bio import BiopythonWarning
 from Bio import Alphabet
 from Bio.Alphabet import IUPAC
-from Bio.Data.IUPACData import ambiguous_dna_complement, ambiguous_rna_complement
+from Bio.Data.IUPACData import (ambiguous_dna_complement,
+                                ambiguous_rna_complement)
```

Comment:
It's valid (PEP328) and PEP8 compliant. I prefer it over having either two separate imports or using a backslash. This is even more useful in `test_seq.py` which has 4 imports from `Bio.Data.IUPACData`